src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1395,6 +1395,51 @@ type PageInfo {
     endCursor: String
 }
 
+type VehiclePosition {
+    """Feed-scoped ID that uniquely identifies the vehicle in the format FeedId:VehicleId"""
+    vehicleId: String,
+    """Human-readable label of the vehicle, eg. a publicly visible number or a license plate"""
+    label: String
+    """Latitude of the vehicle"""
+    lat: Float
+    """Longitude of the vehicle"""
+    lon: Float
+    """The current stop where the vehicle will or is currently arriving."""
+    stop: VehicleStop!
```

Comment:
Should we somehow describe that this is not just the stop with the field name? I really don't know and can't think of a perfect name.

Reply:
I've renamed it to stopRelationship.